environment.yml

```diff
@@ -11,35 +11,119 @@ channels:
 dependencies:
   - pip!=21.3
   - python>=3.8
-  # Python packages that cannot be installed from PyPI:
   - cartopy
-  - compilers
+  - cdo>=1.9.7
+  - cdsapi
+  - cf-units
+  - cftime
+  - cmocean
+  - cython
+  - dask
+  - ecmwf-api-client
+  - eofs
   - esmpy
-  - esmvalcore
+  - esmvalcore 2.5.*
+  - fiona
+  - fire
   - gdal
+  - iris>=3.1.0
+  - jinja2
+  - joblib
+  - lime
+  - mapgenerator>=1.0.5
   - matplotlib-base
-  # Non-Python dependencies
-  - cdo
-  - imagemagick
-  - julia
-  - nco
+  - natsort
+  - nc-time-axis
+  - netCDF4
+  - numpy
+  - numba  # undeclared dependency of xesmf: https://github.com/pangeo-data/xESMF/issues/158
```

Comment:
Wasn't numba removed because of some issues?

Reply:
It looks like the linked issue has been solved, because numba is now listed as a dependency of xesmf: https://github.com/conda-forge/xesmf-feedstock/blob/38d8b192083306b4d7d7d223143fbe683fcb24a5/recipe/meta.yaml#L29

I will remove it here

Reply:
Indeed we removed numba as a dependency of ESMValTool some time ago, because it was not needed and it made updating to a new Python version very slow because we needed to wait for numba to support that new version. However, if our dependencies add it as one of theirs we cannot do much about it.

Reply:
Removed